tools/ci_build/github/linux/docker/scripts/install_server_deps.sh

```diff
@@ -2,9 +2,14 @@
 set -e
 
 SYS_LONG_BIT=$(getconf LONG_BIT)
-
-apt-get remove -y libprotobuf-dev libprotobuf9v5 protobuf-compiler libprotobuf-lite9v5
-apt-get update && apt-get install -y --no-install-recommends libre2-dev
+apt-get update
+#see: https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore21
+if [ "$OS_VERSION" = "16.04" ]; then
```

Comment:
OS_VERSION variable isn't set anywhere.
i think you're missing
OS_VERSION=$(lsb_release -r -s)

Reply:
👍 will fix